### DISCUSSION TITLE
Cheat Sheet JS

### DISCUSSION THREAD

**Discussion by GabrielVillagran** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#336] [Ideas]
![img](https://user-images.githubusercontent.com/44887537/157589939-c1f0d90a-eab2-489b-83fe-adc1dc3933af.png)
Hola que tal explorers, les comparto esta pagina que nos da una cheat sheet sobre JavaScript, espero les sea de utilidad para todo nuestro transcurso y les guste, se explican algunos ejemplos de las funciones del lenguaje y tambien estan las funciones que podemos utilizar de manera resumida :)
https://websitesetup.org/javascript-cheat-sheet/

**Comment by JuanSalvador73** [comment]
Gracias por el aporte :D
